### PR TITLE
Gunset case resprite

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Title
No balance changes. Slight description changes.
Was gonna try my hand at #17278 and make gun cases fit what they initially start with.

Storage is ass.
I wasted hours.

Dont want these sprites to go to waste though, they're real nice :3

_This PR also includes unused "Generic" Gun Cases (with no AD Logo) and Small variants (for sidearms). Neither are implemented, but are there for someone smarter than me to put to use._

## How This Contributes To The Skyrat Roleplay Experience
image

Gun case with actual inserts instead of a black void.... mmhmhgh

(These are the unused ones: )
image

## Proof of Testing
Posted in HTCTTSRE but here it is again i lo veit
image

## Changelog

:cl:
imageadd: Armadyne has released new Gunset Cases with soft foam inserts. They're still struggling to find out how to fit items back into the case, though...
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
Most guns that come in the cases aren't even armadyne guns I like the look of the just generic logo-less one better.

**Comment by ghost** [comment]
> Most guns that come in the cases aren't even armadyne guns I like the look of the just generic logo-less one better.

@Ebin-Halcyon Thoughts? Should I just make the armadyne ones the unused sprites and go with the plain guncases?

**Comment by Ebin-Halcyon** [comment]
yeah just making the base case the main type would be better in the long run.

File: modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm [hunk 1]
```diff
@@ -3,21 +3,25 @@
...
 /obj/item/storage/box/gunset
-	name = "gun supply box"
-	desc = "An Armadyne weapons supply box."
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/gunsets.dmi'
-	icon_state = "box"
-	var/box_state = "box"
-	var/opened = FALSE
+	name = "gun case"
+	desc = "A gun case, with foam inserts laid out to fit a weapon, magazines, and gear securely."
+	icon_state = "guncase_ad" //Currently only comes in Armadyne, though there's sprites for generic ones in the icon file. There's also sprites for smaller ones!
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm]:
```suggestion
	icon_state = "guncase" //Currently only comes as a generic gray, though there's sprites for Armadyne branded ones in the icon file. There's also sprites for smaller ones!
```

File: modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm [hunk 2]
```diff
@@ -3,21 +3,25 @@
...
 	drop_sound = 'sound/items/handling/ammobox_drop.ogg'
 	pickup_sound =  'sound/items/handling/ammobox_pickup.ogg'
 	foldable = FALSE
 	illustration = null
+	var/opened = FALSE
+
+/obj/item/storage/box/gunset/Initialize()
+	. = ..()
+	//Move this to a subtype or something if you make a generic version using the non-armadyne sprites
+	desc += "<br>It has a textured carbon grip, and the <b>[span_red("Armadyne Corporation")]</b> logo etched into the top."
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm]:
```suggestion
//Add this extra line to examine() if you make an armadyne variant: "It has a textured carbon grip, and the <b>[span_red("Armadyne Corporation")]</b> logo etched into the top."
```

File: modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm [hunk 3]
```diff
@@ -3,21 +3,22 @@
...
 
 /obj/item/storage/box/gunset
-	name = "gun supply box"
-	desc = "An Armadyne weapons supply box."
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/gunsets.dmi'
-	icon_state = "box"
-	var/box_state = "box"
-	var/opened = FALSE
+	name = "gun case"
+	desc = "A gun case, with foam inserts laid out to fit a weapon, magazines, and gear securely."
```
Comment (tf-4) [inline modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm]:
```suggestion
	desc = "A gun case with foam inserts laid out to fit a weapon, magazines, and gear securely."
```

**Comment by ghost** [comment]
Realized i hate the gun cases as a concept anyways
Whoops

**Review by tf-4 — APPROVED** [review]
Code seems fine

**Review by Ebin-Halcyon — APPROVED** [review]
Sprites are fine.

**Pull request merged**